vertx-web/src/client/vertx-eventbus.js

```diff
@@ -83,7 +83,9 @@
     this.defaultHeaders = null;
 
     // default event handlers
-    this.onerror = console.error;
+    this.onerror = function (err) {
+      console.error(err);
```

Comment:
I've checked on http://caniuse.com/#search=console.error

I'm a bit afraid by:
"Only supports console functions when developer tools are open, otherwise the console object is undefined and any calls will throw errors."